go.mod

```diff
@@ -8,11 +8,11 @@ require (
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/microerror v0.3.0
 	github.com/giantswarm/micrologger v0.5.0
-	k8s.io/api v0.19.2
-	k8s.io/apiextensions-apiserver v0.19.2
-	k8s.io/apimachinery v0.19.2
-	k8s.io/client-go v0.19.2
-	sigs.k8s.io/controller-runtime v0.7.0
+	k8s.io/api v0.18.9
+	k8s.io/apiextensions-apiserver v0.18.9
+	k8s.io/apimachinery v0.18.9
+	k8s.io/client-go v0.18.9
+	sigs.k8s.io/controller-runtime v0.6.4
```

Comment:
0.7.0 of controlller-runtime broke the build in app-operator. I think we should revert to 0.6.4 until CAPI ship a new version and we can get rid of the forks.


```
# sigs.k8s.io/cluster-api/util/predicates
../../../.gvm/pkgsets/go1.15.3/global/pkg/mod/github.com/giantswarm/cluster-api@v0.3.10-gs/util/predicates/generic_predicates.go:147:91: e.MetaNew undefined (type event.UpdateEvent has no field or method MetaNew)
../../../.gvm/pkgsets/go1.15.3/global/pkg/mod/github.com/giantswarm/cluster-api@v0.3.10-gs/util/predicates/generic_predicates.go:150:88: e.Meta undefined (type event.CreateEvent has no field or method Meta)
../../../.gvm/pkgsets/go1.15.3/global/pkg/mod/github.com/giantswarm/cluster-api@v0.3.10-gs/util/predicates/generic_predicates.go:153:88: e.Meta undefined (type event.DeleteEvent has no field or method Meta)
../../../.gvm/pkgsets/go1.15.3/global/pkg/mod/github.com/giantswarm/cluster-api@v0.3.10-gs/util/predicates/generic_predicates.go:156:89: e.Meta undefined (type event.GenericEvent has no field or method Meta)
FAIL	github.com/giantswarm/app-operator/v3/service/controller/app/resource/chart [build failed]
FAIL	github.com/giantswarm/app-operator/v3/service/controller/app/resource/configmap [build failed]
FAIL	github.com/giantswarm/app-operator/v3/service/controller/app/resource/secret [build failed]
```